docs/README.md

```diff
@@ -1,27 +1,35 @@
 # Store Indexer
 
-An VTEXIO app that listens to IO's broadcasters and indexes data according to catalog changes.
+The Store Indexer is a VTEX IO app designed to listen to IO broadcasters and index data based on catalog changes.
 
-#### Messages
+## Key features
 
-It updates translations when translatable fields changes
+### Messages
 
-#### Routes
+The app updates translations whenever translatable fields are modified.
 
-It updates internal routes. Saves correct page type and id of the canonical routes, and it also saves 
-deactivated products as not found.
+### Routes
+
+Store Indexer updates internal routes by saving the correct page type and canonical route ID, and flags deactivated products as "not found".
 
 ### Search URLs
 
-Indexes the canoncial of the top searches 
+Store Indexer indexes the canonical URLs of top searches.
+
+### Brand Search as Full Text
+
+Enhance **Intelligent Search** by resolving brand search queries as full text. This configuration enables search results to include products where the brand name appears in fields like the product title or description, rather than limiting results to a brand filter. For detailed instructions, refer to [Setting up brand search queries as full text](https://developers.vtex.com/docs/guides/setting-up-brand-search-queries-as-full-text).
```

Comment:
```suggestion
- **Brand search as full text:** Enhances [Intelligent Search](https://help.vtex.com/pt/tracks/vtex-intelligent-search--19wrbB7nEQcmwzDPl1l4Cb) by resolving brand search queries as full text. This configuration enables search results to include products where the brand name appears in fields like the product title or description, rather than limiting results to a brand filter. For detailed instructions, refer to [Setting up brand search queries as full text](https://developers.vtex.com/docs/guides/setting-up-brand-search-queries-as-full-text).
```